### ISSUE TITLE
Holy Sites - missing/changes

### ISSUE THREAD

**Issue by rquinio** [loup99/WtWSMS#44]
Some holy sites are missing for some religions, breaking their gameplay. Currently we have:
georgian_pagan (1/5)
scythian_pagan / scythian_pagan_reformed (4/5)
bosporan_faith (0/5)

DONE:
docetic (1/5)
kemetic_pagan (1/5)
manichean (2/5)
semetic_pagan / semetic_pagan_reformed (3/5)

**Comment by rquinio** [comment]
For info, for kemetic_pagan, the ones used in CK2+ are:
b_zweitina
b_ikonion
b_fuwa
b_memphis
b_luxor (temple holding needs to be re-constructed)

![kemetic_holy_sites](https://cloud.githubusercontent.com/assets/1886883/5695468
/915cc216-99a5-11e4-8cea-3c12c2fafac9.jpg)

**Comment by loup99** [comment]
We won't do the same for kemetic, it does not fit.

**Comment by loup99** [comment]
What we wanted to have for Kemetic: the capital of the Garamantes, Memphis, Thebes, Giza, and Abydos. One of those might be replaced by b_luxor.

**Comment by rquinio** [comment]
Ok, makes sense to adapt to the time period and gameplay balance !
In fact in game Luxor and Thebes are the same thing (b_luxor is one of the temple of c_asyut)
I don' think Giza exist as barony, as it's very close to Memphis. Also Abydos deson't seem to exist.

**Comment by loup99** [comment]
There is some Abydos in Egypt (not sure where) Giza can be modified with something accurate, but not egyptian.

**Comment by rquinio** [comment]
Found out Abydos (b_egypabydos), but it's a city, and in same county as temple b_luxor.

**Comment by loup99** [comment]
Oh, better to use that then.

**Comment by loup99** [comment]
For the Insular Holy sites, this suggestion is a lot better than the current setup: http://forum.paradoxplaza.com/forum/showthread.php?772971-When-the-World-Stopped-Making-Sense-A-480-Migrational-Period-Mod&p=17812706&viewfull=1#post17812706

**Comment by loup99** [comment]
Changed this title as this discussion can also be used to change some existing ones.

**Comment by rquinio** [comment]
From error.log:
````
[religion.cpp:111]: Not enough Holy Sites scripted for religion: Aquileian
[religion.cpp:111]: Not enough Holy Sites scripted for religion: Islamic
[religion.cpp:111]: Not enough Holy Sites scripted for religion: Abadites
[religion.cpp:111]: Not enough Holy Sites scripted for religion: Quranist
[religion.cpp:111]: Not enough Holy Sites scripted for religion: Qadariyya
[religion.cpp:111]: Not enough Holy Sites scripted for religion: Khawariji
[religion.cpp:111]: Not enough Holy Sites scripted for religion: Haruriyyah
[religion.cpp:111]: Not enough Holy Sites scripted for religion: Sanamahi
[religion.cpp:111]: Not enough Holy Sites scripted for religion: Armazi
[religion.cpp:111]: Not enough Holy Sites scripted for religion: Eiateuta
[religion.cpp:111]: Not enough Holy Sites scripted for religion: Wodenic
[religion.cpp:111]: Not enough Holy Sites scripted for religion: Tabitian
[religion.cpp:111]: Not enough Holy Sites scripted for religion: Tabitian
[religion.cpp:111]: Not enough Holy Sites scripted for religion: Droemnät
[religion.cpp:111]: Not enough Holy Sites scripted for religion: Cybelian
[religion.cpp:111]: Not enough Holy Sites scripted for religion: Hypsistarian
```

**Comment by loup99** [comment]
Most of these holy sites are quiet daunting and impossible to find data on. How should we act in these cases? Wait upon the return of Enlil? Try to find the most balanced spots? Ask the players?

**Comment by rquinio** [comment]
Yeah, maybe we could ask player's help.

**Comment by EWattman** [comment]
Well, until Enlil returns, Im in command of the mod and I think we can't just sit and wait for his return. We need to move forward, even without him. Definetly ask for player's help.

**Comment by loup99** [comment]
I asked in the main thread, we will have to see.

**Comment by Arcvalons** [comment]
Honestly, are some of those super-obscure religions even necessary ?

**Comment by loup99** [comment]
Yes, we are not removing any religions.

**Comment by Arcvalons** [comment]
Okay, looked about some of these religions, and I believe Cybelian and Hypsistarian could probably be Hellenic heresies (with mostly the same Holy Sites, maybe one moved to Israel for Hypsistarians) , and Hellenic itself could become its own religious group. Some other existing religions, like Promethean, would also fit nicely within an Hellenic group.

**Comment by rquinio** [comment]
Well, it's not really a problem to have multiple mainstream religions (with multiple heresies) in the same religion group (pagan group). I think the idea is that polytheistic religions were much more tolerant to other religions than the monotheistic ones.
What would be the benefit of creating a separate religion group for Hellenic ?

**Comment by Arcvalons** [comment]
Organization, pretty much. I think it could help in the long run to differantiate hellenic religion/civilization separately from, say, arab or finnic paganism, in the code. Plus, hellenic is much bigger and important in 479 than in 769, with several more spin-offs, cults/heresies, etc. branching from it still alive, and they did see themselves as more civilized than others such as the germanics.

**Comment by loup99** [comment]
Hellenic is not especially significant at start, compared to other faiths. Germanic deserves a separated group if anything in that case. I don't think there is an advantage.

**Comment by Arcvalons** [comment]
Yeah, I was going to say the same argument could be used for Germanic. Anyway, right now Cybelian is its own separate Pagan faith and Hypsistarian has a group for itself; Cybele was incorporated into the Hellenic pantheon in 400 BCE so it'd fit as a Hellenic heresy, and the Hypsistarian... were strange, we only have greek sources talking about them, but they seem similar to Jews. Either a Judaism heresy or an Hellenic one.

**Comment by CaudilloFranco** [comment]
Is there Hellenic Judaism in-game? That would be a perfect parent religion to branch off of, according to Wikipedia, hell, I think it should be made as its own new group or at least under the same religious group as Judaism, since Hellenic Judaism was a blend of both IRL. If not they should be Jewish in my opinion.

Date: Tue, 8 Dec 2015 07:31:52 -0800

**Comment by loup99** [comment]
First of all: this discussion is only about holy sites, not religions in general. 

Secondly, I don't see why we should split to split, while adding virtually no content. Vanilla has it in a way for a reason. Splitting off the Germanic religions doesn't give us much, but we loose all of the effects of the pagan group. 

Hellenic is a minority religion in the mod, not the faith it was before (although I would like to argue that Hellenism on its own has never been significant due to its nature). With the timeframe we have, and even the planned one, I don't see much need for adding Helleno-Judaism, as it would represent no provinces ingame.

But what I do think is that, in parallel with the wiki article, we should make a tree that lists all religions and branches, so that we can look  at changes or splits more easily and get different sources together, concerning Gnosticism, Manicheanism, Hypsistarian and etc. That way we could uniformise the whole tree and look at what features we want and which we do not, as well as what changes make he most sense. (this would also help with church councils)

**Comment by rquinio** [comment]
Thanks to the work of @GrandiSlayer , all religions have 5 holy sites ! So closing this one.
